src/pages/send/multi-send/multi-send.ts

```diff
@@ -224,8 +226,9 @@ export class MultiSendPage extends WalletTabsChild {
         recipient
       });
     } else {
+      const address = this.incomingDataProvider.extractAddress(this.search);
       const newRecipient = {
-        toAddress: _.clone(this.search),
+        toAddress: _.clone(address),
```

Comment:
Currently if you paste a btc or bch addresses with specific amount, like:
`bitcoin:39J94u1DaGVimKG8YSsrYyEaTc4ioLEwwT?amount=0.02000000`
`bitcoincash:pr95tshknyl0drk5sp538vz6gul2vvd2vu080xzej9?amount=0.01000000`
 the recipient is added with that amount without taking you to amount page. Maybe we can do the same for eth and `value:`
`ethereum:0xDF5C0dd7656bB976aD7285a3Fb80C0F6B9604576?value=3000000000000000000`

Reply:
Changed to parse the value/amount properly for EthereumUri.

If value or amount is missing for any URI, redirects to amount page.